smartsim/_core/mli/infrastructure/control/workermanager.py

```diff
@@ -161,12 +168,13 @@ class WorkerManager(Service):
 
     def __init__(
         self,
-        task_queue: "mp.Queue[bytes]",
+        file_like_interface: "fli.FLInterface",
```

Comment:
We should consider a swap to injecting an instantiated `CommChannelBase` instantiation to avoid passing in the `FLInterface` separately.

This will also avoid requiring a dragon environment to test worker manager

Reply:
Done!